crt_portal/cts_forms/views.py

```diff
@@ -916,6 +916,21 @@ def _notification_change(request):
     return redirect(reverse('crt_forms:crt-forms-notifications'))
 
 
+@login_required
+def resources_view(request):
+    resources = Resource.objects.all()
+    data = []
+    for index, resource in enumerate(resources):
+        data.append({
+            'resource': resource,
+            'tags': list(map(str, resource.tags.values_list('name', flat=True))),
```

Comment:
you may want to "prefetch related" on these tags

You can check the admin toolbar's SQL section to confirm, but this may be doing one query per each resource:

```sql
SELECT name
FROM tags
WHERE resource = etc
```

[prefetch_related](https://docs.djangoproject.com/en/5.1/ref/models/querysets/#prefetch-related) tries to make that a single query instead of many